src/common/droid-util.c

```diff
@@ -70,35 +71,44 @@
 #include "droid/droid-config.h"
 #include "droid/conversion.h"
 #include "droid/sllist.h"
+#include "droid/utils.h"
 
-struct droid_quirk {
+struct droid_option {
     const char *name;
     uint32_t value;
 };
 
-struct droid_quirk valid_quirks[] = {
-    { "input_atoi",             QUIRK_INPUT_ATOI            },
-    { "set_parameters",         QUIRK_SET_PARAMETERS        },
-    { "close_input",            QUIRK_CLOSE_INPUT           },
-    { "unload_no_close",        QUIRK_UNLOAD_NO_CLOSE       },
-    { "no_hw_volume",           QUIRK_NO_HW_VOLUME          },
-    { "output_make_writable",   QUIRK_OUTPUT_MAKE_WRITABLE  },
-    { "realcall",               QUIRK_REALCALL              },
-    { "unload_call_exit",       QUIRK_UNLOAD_CALL_EXIT      },
-    { "output_fast",            QUIRK_OUTPUT_FAST           },
-    { "output_deep_buffer",     QUIRK_OUTPUT_DEEP_BUFFER    },
-    { "audio_cal_wait",         QUIRK_AUDIO_CAL_WAIT        },
-    { "standby_set_route",      QUIRK_STANDBY_SET_ROUTE     },
-    { "speaker_before_voice",   QUIRK_SPEAKER_BEFORE_VOICE  },
+struct droid_option valid_options[] = {
+    { "input_atoi",             DM_OPTION_INPUT_ATOI            },
+    { "close_input",            DM_OPTION_CLOSE_INPUT           },
+    { "unload_no_close",        DM_OPTION_UNLOAD_NO_CLOSE       },
+    { "hw_volume",              DM_OPTION_HW_VOLUME             },
+    { "output_make_writable",   DM_OPTION_OUTPUT_MAKE_WRITABLE  },
+    { "realcall",               DM_OPTION_REALCALL              },
+    { "unload_call_exit",       DM_OPTION_UNLOAD_CALL_EXIT      },
+    { "output_fast",            DM_OPTION_OUTPUT_FAST           },
+    { "output_deep_buffer",     DM_OPTION_OUTPUT_DEEP_BUFFER    },
+    { "audio_cal_wait",         DM_OPTION_AUDIO_CAL_WAIT        },
```

Comment:
audio_cal_wait, standby_set_route, speaker_before_voice are not needed with Android 11 adaptations (at least for the moment).